modules/filtering-insights-advisor-recommendations.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="filtering-unnecessary-advisor-recommendations_{context}"]
+= Filtering Insights advisor recommendations
+
+As an {product-title} cluster manager, you can filter the recommendations that are displayed on the recommendations list. By applying filters, you can reduce the number of reported recommendations and concentrate on your highest priority recommendations.
+
+The following procedure demonstrates how to set and remove *Category* filters; however, the procedure is applicable to any of the filter types and respective values. 
+
+.Prerequisites
+You are logged in to the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console].
+
+.Procedure
+. Navigate to *Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*.
```

Comment:
```suggestion
. Go to *Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*.
```
Current [IBM Style word usage guidelines](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n) say to use _navigate_ "only when a more direct verb, such as _click_, _select_, _browse_, or _go to_, is not applicable".

Reply:
Done